docs/catalog/overview.md

```diff
@@ -4,8 +4,7 @@ file. -->
 
 # Datasets
 
-Note: The datasets documented here are from `HEAD` and so not all are available
-in the current `tensorflow-datasets` package. They are all accessible in our
+Note: The datasets marked with `nights-stay` are available only in the TFDS
```

Comment:
The `nights-stay` icon should be displayed, not the name. See md source code to view how to display the `nights-stay` icon (https://iconify.design/icon-sets/ic/round-nights-stay.html)

Reply:
Fixed